Airbnb | Homes

samat1989 opened the pull request:
- Рефакторинг компонентов по бизнес-логике 
- Компоненты и изображения раскиданы по директориям где используются 
- директория images удалена 
- не приложил now.sh потому что пока был только рефакторинг

evgenyrodionov commented on src/index.js:
```diff
@@ -3,7 +3,6 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import registerServiceWorker from "./registerServiceWorker";
-import "normalize.css";
```
почему удалил?

samat1989 commented:
- refactoring 
- added mobile, tablet and desktop view for Homes 
- added map 
- have some problems with header-like-figma-looking, I'll try to fix it tommorow

https://build-czwqxdkuum.now.sh/

evgenyrodionov commented:
Далеко от макета

![image](https://user-images.githubusercontent.com/775692/32902552-bc9573fa-cb03-11e7-9a9e-725f4b07193c.png)
![image](https://user-images.githubusercontent.com/775692/32902557-bf0b87aa-cb03-11e7-8d4d-4bd45d0c44bb.png)

![image](https://user-images.githubusercontent.com/775692/32902567-c4b36f2e-cb03-11e7-966d-0204e15bc88e.png)
![image](https://user-images.githubusercontent.com/775692/32902570-c6d6733c-cb03-11e7-859a-9bd71e655fa4.png)

![image](https://user-images.githubusercontent.com/775692/32902584-d18d5606-cb03-11e7-9f37-87d3830c7eb7.png)

![image](https://user-images.githubusercontent.com/775692/32902579-ce2e75a8-cb03-11e7-86e4-f247a0b522a6.png)

evgenyrodionov commented:
![image](https://user-images.githubusercontent.com/775692/32902761-5212d9cc-cb04-11e7-9731-f6deafd30862.png)

![image](https://user-images.githubusercontent.com/775692/32902761-5212d9cc-cb04-11e7-9731-f6deafd30862.png)

samat1989 commented:
- corrected styles of main and homes pages 
- added normalize, 
- in Homes corrected map view, added map button for tablet and mobile 

 https://build-hdhfudowwt.now.sh

evgenyrodionov commented:
Шрифты в хедере не те
![image](https://user-images.githubusercontent.com/775692/32947905-4b873b16-cbae-11e7-89a2-073b710decf9.png)

у заголовков цвет не #000


В карточках шрифты тоже не те
![image](https://user-images.githubusercontent.com/775692/32947925-5cae448e-cbae-11e7-82b5-f45d52b8d605.png)

Отступы у футера, шрифты
![image](https://user-images.githubusercontent.com/775692/32947928-61eee886-cbae-11e7-9fca-b569bb9d351b.png)

Отступы в карточке, отступы в фильтрах, карта не уходит в правый край, шрифты в фильтрах
![image](https://user-images.githubusercontent.com/775692/32947952-752b1c26-cbae-11e7-8683-68bbb297306f.png)

evgenyrodionov commented:
Логотип не кликабельный, реакт-роутеровский Link не используешь

evgenyrodionov commented on src/Header.js:
```diff
@@ -3,13 +3,14 @@ import logo from "./logo.svg";
 import styled from "styled-components";
 import search from "./search.svg";
 
+import { Inner, FlexWrapper } from "./UI.js";
```
.джс опусти

названия компонентов ужасные и неочевидные

samat1989 commented:
- Рефакторинг фильтров
- Добавил реакт-роутеровский Link 
- Поправил шрифты, отступы, цвета. Теперь всё должно быть ок.  

https://build-tnfncpnzsj.now.sh

evgenyrodionov commented:
Шрифт не тот
![image](https://user-images.githubusercontent.com/775692/32979532-dd51ad24-cc67-11e7-9db7-737c6cfdba87.png)

Отступы, контент
![image](https://user-images.githubusercontent.com/775692/32979535-e70b5e64-cc67-11e7-8d46-2958a7651dc0.png)

Отступы
![image](https://user-images.githubusercontent.com/775692/32979537-ebf9e738-cc67-11e7-8667-6e9e1255bc7c.png)

Опять же шрифты
![image](https://user-images.githubusercontent.com/775692/32979548-11496fb8-cc68-11e7-9cd1-33a155bc5bf1.png)

Карта куда-то вниз уходит
![image](https://user-images.githubusercontent.com/775692/32979550-1b4f0c8e-cc68-11e7-9a42-04a63c17e233.png)

Шрифты, отступы, контент
![image](https://user-images.githubusercontent.com/775692/32979553-2116fa96-cc68-11e7-8393-f1d588ce2859.png)

Фильтр нерабочий
![image](https://user-images.githubusercontent.com/775692/32979555-27ff06b4-cc68-11e7-8286-d797a0c4813b.png)

samat1989 commented:
Про шрифты на фильтре не совсем понял. 

<img width="1201" alt="2017-11-18 15 45 51" src="https://user-images.githubusercontent.com/24860358/32980452-a87d7aa0-cc77-11e7-90a2-0131997ce82d.png">

samat1989 commented:
https://build-ekxweiljou.now.sh

evgenyrodionov commented on .env:
```diff
@@ -0,0 +1 @@
+var REACT_APP_GOOGLE_MAPS_KEY = AIzaSyBSj3eUnDv6MTESPGwUCb3j9yTBeROicuA
```
вар? зачем?

evgenyrodionov commented on =#.##:
```diff
@@ -0,0 +1,11 @@
+yarn add v1.3.2
+[1/4] Resolving packages...
+[2/4] Fetching packages...
```
чё за мусор

evgenyrodionov commented on src/Homes/Filters/Button.js:
```diff
@@ -0,0 +1,82 @@
+
+  render() {
+    if (this.state.label === null) {
+      this.setState({ label: this.props.label });
```
в рендере сетстейт нельзя вызывать

evgenyrodionov commented on src/Homes/Filters/Button.js:
```diff
@@ -0,0 +1,82 @@
+  };
+
+  selectDates = (start, end) => {
+    this.setState({
```
кошмар

evgenyrodionov replied:
нахуй тебе это всё в классе? вынеси в отдельную функцию, передавай ей параметры и возвращай значения

samat1989 commented:
refactoring and fixes 
https://build-hmzyydmxfy.now.sh/

evgenyrodionov commented on .env:
```diff
@@ -1 +1 @@
-var REACT_APP_GOOGLE_MAPS_KEY = AIzaSyBSj3eUnDv6MTESPGwUCb3j9yTBeROicuA
\ No newline at end of file
+REACT_APP_GOOGLE_MAPS_KEY = AIzaSyBSj3eUnDv6MTESPGwUCb3j9yTBeROicuA
```
не храни кредс в репозитории

evgenyrodionov commented on src/Homes/Filters/Guests.js:
```diff
@@ -0,0 +1,25 @@
+
+import Wrapper from "../../UI";
+
+onToggle = isSelected => {
```
ты проебал const

evgenyrodionov replied:
более того, это вообще почему-то не в классе, хотя дальше идёт обращение как к методу класса

evgenyrodionov commented on src/Homes/Filters/index.js:
```diff
@@ -11,45 +10,22 @@ const Filter = styled.div`
 export default Filters;
+
+//  <Button label="Guests" />
+//         <Button label="Room type" />
```
мусор

evgenyrodionov commented:
И выкинь этот мусор 

https://github.com/samat1989/airbnb-demo-frontend/pull/3/files/a974baf5864f7a32ed7a7b9ecd075c83af1dc409..a5de2190c877200eac2c8e67b2d0a7862f2bc089#diff-cc80886b40760f9465c0f2ca79c37d39

samat1989 commented:
- Добавил метод map в Homes.
- Вроде бы убрал env в gitignore 
- Создал компоненты остальных фильтров, только пока требуют достилизации и не работают. 
- Про этот мусор, не совсем понимаю где это смотреть и как его удалить https://github.com/samat1989/airbnb-demo-frontend/pull/3/files/a974baf5864f7a32ed7a7b9ecd075c83af1dc409..a5de2190c877200eac2c8e67b2d0a7862f2bc089#diff-cc80886b40760f9465c0f2ca79c37d39
https://build-ojawffwaoz.now.sh

evgenyrodionov commented on .gitignore:
```diff
@@ -20,4 +21,4 @@ npm-debug.log*
 yarn-error.log*
 
-*.map
+.map*
```
не понял, зачем поменял?
